cypress/integration/rich-text/RichTextEditor.Pasting.spec.ts

```diff
@@ -274,6 +274,22 @@ describe(
       });
     });
 
+    describe('Superscript and subscript marks', () => {
+      it('works when pasting subscript and superscript from a google doc', () => {
```

Comment:
The test title was updated but the snapshot remained unchanged. I think you have to do that manually

Reply:
Great catch! sorry about that